src/ontology/cl-edit.owl

```diff
@@ -29018,6 +29020,17 @@ SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P60041))
 
+# Class: obo:CL_4023026 (direct pathway medium spiny neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Medium_spiny_neuron&oldid=1109726850"^^xsd:string) obo:IAO_0000115 obo:CL_4023026 "A medium spiny neuron that expresses dopamine type 1 receptors and projects to the globus pallidus internus or the substantia nigra pars reticulata."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023026 <https://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasExactSynonym obo:CL_4023026 "D2-MSN")
```

Comment:
This is a synonym for indirect pathway MSN.

Reply:
argh sorry, did the correct subclass but somehow messed up the synonym >.<